### PR TITLE
chore(docs): add map

### PR DESCRIPTION
**Pull request by Stamo-Gochev**
Add map spec.

### CODE REVIEW

File: docs/accessibility/Map.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Splitter
```
Comment (joneff) [inline docs/accessibility/Map.md:2]:
```suggestion
title: Map
```

File: docs/accessibility/Map.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+category: accessibility
+slug: splitter
```
Comment (joneff) [inline docs/accessibility/Map.md:4]:
```suggestion
slug: map
```

**Review by joneff — CHANGES_REQUESTED** [review]

File: docs/accessibility/Map.md (hunk 1 above)
Comment (Stamo-Gochev) [reply docs/accessibility/Map.md]: Fixed in https://github.com/telerik/kendo-themes/pull/3214/commits/f1e1d9fb4d1f3e202c270b5446538a6fe84db2f7

File: docs/accessibility/Map.md (hunk 2 above)
Comment (Stamo-Gochev) [reply docs/accessibility/Map.md]: Fixed in https://github.com/telerik/kendo-themes/pull/3214/commits/f1e1d9fb4d1f3e202c270b5446538a6fe84db2f7

**Comment by Stamo-Gochev** [comment]
The position of the articles is also regenerated in https://github.com/telerik/kendo-themes/pull/3214/commits/a0940437c5d53a439e261fda2246d5db8b26725b

**Review by joneff — APPROVED** [review]

**Comment by Stamo-Gochev** [comment]
Merging is still blocked for me
image

**Pull request merged**